docs/src/main/sphinx/release/release-475.md

```diff
@@ -0,0 +1,61 @@
+# Release 475 (16 Apr 2025)
+
+## General
+
+* Publish additional metrics for input tables in event listener. ({issue}`25475`)
+* Expose dynamic filter statistics in the `QueryCompletedEvent`. ({issue}`25575`)
+* Improve scalability of inline data encoding in the parallel client protocol. ({issue}`25439`)
```

Comment:
@wendigo, what does "scalability" mean in this context?

Reply:
Previously the inlined segment was produced on the coordinator meaning that Pages were converted to JSON and eventually compressed there. Now, the inline segment is encoded and compressed on the worker nodes and coordinator is just passing bytes back to the client. This reduces the amount of CPU used on the coordinator when data is being inlined.

Reply:
In the docs we still call it the spooling protocol..